app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command, first_file, second_file = command.split(" ")
```

Comment:
`Split()` method by default splitting by `" " value `, so you can just write `split()`